aksel.nav.no/website/pages/templates/soknad-introside-aap/soknad-introside-aap.tsx

```diff
@@ -0,0 +1,314 @@
+  BodyShort,
+  Box,
+  Button,
+  Checkbox,
+  GuidePanel,
+  Heading,
+  Link,
+  List,
+  Page,
+  Show,
+  Stack,
+  VStack,
+} from "@navikt/ds-react";
+
+function Example() {
+  useDekorator();
+
+  return (
+    <Page data-aksel-template="form-intropage-v1" footer={<Footer />}>
+      <Header />
+      <VStack as="main" gap="8">
+        <Page.Block width="text" gutters>
+          <Bleed marginInline={{ lg: "24" }}>
+            <Stack
+              align="center"
+              gap="6"
+              direction={{ xs: "row-reverse", lg: "row" }}
+              justify="start"
+              wrap={false}
+            >
+              <Show above="sm">
+                <ApplicationPictogram title="Application" />
+              </Show>
+              <VStack gap="1">
+                <BodyShort size="small">NAV 10-07.03 (om relevant)</BodyShort>
+                <Heading level="1" size="large">
+                  Søknad om arbeidsavklaringspenger (AAP)
+                </Heading>
+              </VStack>
+            </Stack>
+          </Bleed>
+        </Page.Block>
+        <Page.Block width="text" gutters>
+          <VStack gap="12">
+            <GuidePanel poster>
+              <Heading level="2" size="medium" spacing>
+                Hei, Luke Skywalker!
+              </Heading>
+              <BodyLong spacing>
+                Jeg er her for å veilede deg gjennom søknaden. Svarene dine
+                lagres underveis, slik at du trygt kan gå tilbake og endre dem.
+              </BodyLong>
+              <BodyLong spacing>
+                Arbeidsavklaringspenger forkortes ofte med “AAP”.
+              </BodyLong>
+              <BodyLong>
+                AAP skal sikre deg inntekt når du på grunn av sykdom eller skade
+                har behov for å få avklart mulighetene dine til å jobbe.{" "}
+                <Link href="#">
+                  Les mer om AAP, hvem som kan søke og hva du kan få på nav.no
+                </Link>
+                .
+              </BodyLong>
+            </GuidePanel>
+            <div>
+              <Heading level="2" size="large" spacing>
+                Før du søker
+              </Heading>
+              <List>
+                <List.Item>
+                  Vanligvis kan du tidligst få AAP fra den dagen du søker.
+                </List.Item>
+                <List.Item>
+                  Du kan søke om AAP selv om du ikke har mottatt sykepenger.
+                </List.Item>
+              </List>
+            </div>
+            <div>
+              <Accordion>
+                <Accordion.Item>
+                  <Accordion.Header>
+                    Informasjon vi henter om deg
+                  </Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong>
+                      Her skal det så informasjon om hvor vi vil hente
+                      opplysninger om søkeren og hva slags opplysninger vi
+                      henter.
```

Comment:
Is the accordion content not supposed to be tailored to the example? (E.g. AAP)

Reply:
No, we don't have example text for this.